internal/connector/internal/services/connector_namespaces.go

```diff
@@ -478,16 +473,16 @@ func (k *connectorNamespaceService) DeleteNamespaceAndConnectorDeployments(ctx c
 	// set connector desired state to connectorDesiredState and status to "deleting" to remove from namespaces
 	if err := dbConn.Where("deleted_at IS NULL AND id IN ?", connectorIds).
 		Updates(&dbapi.Connector{DesiredState: connectorDesiredState}).Error; err != nil {
-		return false, false, services.HandleUpdateError("Connector", err)
+		return count, services.HandleUpdateError("Connector", err)
 	}
 	if err := dbConn.Where("deleted_at IS NULL AND id IN ?", connectorIds).
 		Updates(&dbapi.ConnectorStatus{Phase: dbapi.ConnectorStatusPhaseDeleting}).Error; err != nil {
-		return false, false, services.HandleUpdateError("Connector", err)
+		return count, services.HandleUpdateError("Connector", err)
 	}
 	// mark all deployment statuses as "deleting"
 	if err := dbConn.Where("deleted_at IS NULL AND id IN ?", connectorIds).
 		Updates(&dbapi.ConnectorDeploymentStatus{Phase: dbapi.ConnectorStatusPhaseDeleting}).Error; err != nil {
```

Comment:
Good catch, the query is wrong. Which effectively doesn't actually update the deployment status. I'll fix it.

Reply:
It was a non-issue and didn't break anything though, since the deployment status is overwritten by the agent anyway.

Reply:
Since it's redundant, I'm going to remove that update instead of having to join across two tables and updating something that's going to get overwritten.